USAGE_desktop.md

```diff
@@ -567,6 +567,47 @@ Optional arguments:
   --version             Print version information and exit.
 ```
 
+### JSON Lines Conversion
+
+The `gfxrecon-convert` tool converts a capture file into a series of JSON
+documents, one per line following the
+[JSON Lines standard](https://jsonlines.org/).
+The JSON document on each line is designed to be parsed by tools such as simple
+Python scripts as well as being useful for inspection by eye after pretty
+printing, for example by piping through a command-line tool such as
+[`jq`](https://stedolan.github.io/jq/).
+For these post-processing use cases, `gfxrecon-convert` can be used to stream
+binary captures directly into `stdin` of the next tool in the chain, without
```

Comment:
`binary captures` -> `JSON representations of captured data`?

Reply:
fixed.